src/lib/core/a11y/list-key-manager.ts

```diff
@@ -63,6 +64,9 @@ export class ListKeyManager<T extends CanDisable> {
       case END:
         this.setLastItemActive();
         break;
+      case ESCAPE:
```

Comment:
Yeah, agree that this doesn't seem like a natural fit for a "list" key manager. I like the idea of having a PopupKeyManager, but inheriting from ListKeyManager doesn't seem like quite the right abstraction since sometimes we'll have popups that aren't lists. Maybe it could be a sibling?  Though it's annoying to have to instantiate both a ListKeyManager and a PopupKeyManager. 

I think it could still be useful to have a service for this (rather than keeping in components) because turning the key events into observables makes it easy to combine and pass them around later.